web/components/feed/feed-items.tsx

```diff
@@ -834,6 +857,37 @@ function FeedAnswerGroup(props: {
   )
 }
 
+function FeedGeneralComments(props: {
```

Comment:
Possibly move into contract tabs as a regular component that has ContractActivity as a child